### DISCUSSION TITLE
internal network on utm

### DISCUSSION THREAD

**Discussion by 4lifeonmars** [utmapp/UTM#3988] [UTM Help (macOS)]
Hi to everyone,I'm trying to create an internal environment between two VMs to do some tests, on virtualbox it was very simple because there was the internal network function but on utm I don't know how to configure it. has anyone already done it?

**Comment by osy** [comment]
On 3.2.x you can try setting both VMs to "host only" networking. I haven't tried it myself though.

**Comment by nightman68** [comment]


**Reply by pomozoff** [reply]
> It works with "Bridged (Advanced)" for me

It works, but this is not the solution for the question.
The network must be isolated from outside, since the `Bridged` is shared to the whole subnet it is based on.

**Reply by ktprograms** [reply]
See https://github.com/utmapp/UTM/discussions/3988#discussioncomment-2697849
